jaeger-2.yaml

```diff
@@ -1,23 +1,25 @@
 package:
-  name: jaeger
-  version: "1.74.0"
-  epoch: 0 # CVE-2025-47910
+  name: jaeger-2
+  version: "2.10.0"
+  epoch: 0
   description: Jaeger, a Distributed Tracing Platform
   copyright:
     - license: Apache-2.0
+  dependencies:
+    provides:
+      - jaeger=${{package.full-version}}
 
 pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/jaegertracing/jaeger
       tag: v${{package.version}}
-      expected-commit: c17f97d4add9f96e52852f8f238689568e8a48d9
+      expected-commit: 076ea6f40af6b3f9f75a86caa23972fc0b99ca71
       recurse-submodules: true
 
   - working-directory: jaeger-ui
-    uses: patch
-    with:
-      patches: ../fix-build-react-icons-typescript-errors.patch
+    runs: |
+      sed -i '/react-icons/s/: "\^5\.0\.1"/: "5.0.1"/' packages/plexus/package.json
```

Comment:
This would result build failure due to conflict in the future; so we could use the npm "resolutions" to bump the deps (there are some example)